pom.xml

```
@@ -178,7 +178,7 @@
         <skipUT>false</skipUT>
         <skipIT>true</skipIT>
         <elasticsearch>7</elasticsearch>
-        <slf4j.version>1.7.25</slf4j.version>
+        <slf4j.version>1.7.36</slf4j.version>
```

slf4j-log4j12-1.7.36 will not include log4j.
```suggestion
        <slf4j.version>1.7.25</slf4j.version>
```